.circleci/config.yml

```diff
@@ -170,21 +173,25 @@ jobs:
     steps:
       - checkout
       - run:
-          name: ensure that every template builds for a variety of options.
+          name: Ensure that every template builds for a variety of options.
           command: |
-                  go run u-root.go -build=bb minimal
-                  go run u-root.go minimal
-                  go run u-root.go -build=bb core
-                  go run u-root.go core
-                  go run u-root.go -build=bb coreboot-app
-                  go run u-root.go coreboot-app
-                  go run u-root.go -build=bb all
-                  go run u-root.go all
-                  go run u-root.go -build=bb all core
-                  go run u-root.go all core
-                  go run u-root.go -fourbins minimal
+                  goruncmd="go run u-root.go templates.go -stats-output-path=/tmp/stats.json"
+                  $goruncmd -build=bb minimal
+                  $goruncmd -build=bb core
+                  $goruncmd -build=bb coreboot-app
+                  $goruncmd -build=bb all
+                  $goruncmd -build=bb world
+                  # Fails because of dups. Is this supposed to work?
+                  #$goruncmd -build=bb all core
+                  #$goruncmd all core
+                  go run u-root.go templates.go -fourbins minimal
                   go build .
-                  GOOS=plan9 ./u-root plan9 -defaultsh=/bbin/rush
+                  # Plan9 build is broken.
```

Comment:
broken? Ron's been making it work... got a log?

Reply:
actually, you're right, i got confused there: `GOOS=plan9 go build.` doesn't work, but `GOOS=plan9 ./u-root`. turned it back on.

Reply:
can you turn it back on then?

Reply:
sorry, i fail at git, apparently - pushed the wrong thing. PTAL.

Reply:
@hugelgupf ping